content/crutchcorn/collections/art-of-accessibility/posts/art-of-a11y-preface/index.md

```diff
@@ -0,0 +1,330 @@
+        border-radius: 1rem;
+        padding: 0rem 4rem;
+        background: rgb(98, 0, 238);
+        color: rgb(255, 255, 255);
+        box-shadow: 0px 0.75rem 0.25rem -0.5rem rgba(0, 0, 0, 0.2),
+          0px 0.5rem 0.5rem 0px rgba(0, 0, 0, 0.14),
+          0px 0.25rem 1.25rem 0px rgba(0, 0, 0, 0.12);
+      "
+    >
+      Buy Now
+    </button>
+```
+
+ <button aria-label="A large button with dark a rounded purple background, white 'buy now' text, and a shadow"  style=" height: 9rem; font-size: 4rem; border: medium none; border-radius: 1rem; padding: 0rem 4rem; background: rgb(98, 0, 238); color: rgb(255, 255, 255); box-shadow: 0px 0.75rem 0.25rem -0.5rem rgba(0, 0, 0, 0.2), 0px 0.5rem 0.5rem 0px rgba(0, 0, 0, 0.14), 0px 0.25rem 1.25rem 0px rgba(0, 0, 0, 0.12); " > Buy Now </button>
+
+This button is more rounded, has a drop shadow, and matches Google's design language.
+
+Between the three buttons, which one do you want to use on your page?
+
+The first button is clearly not a good contender, since it's invisible to the end user. That said; between the last two buttons, I think a reasonable argument could be made for either of them. There's few "objectively" correct guidelines for what a button _should_ look like.
+
+It's important, however, to acknowledge that your decisions on the UX can impact your users long into their usage. We should be sure to test our behaviors and verify with those users that they enjoy using your app.
+
+> Why are you talking about this now? How does it matter?
+
+It matters because if the end users don't enjoy the experience of working with our apps, we have some work to do.
+
+One area which many developers have work to do is "Accessibility".
+
+# What is accessibility?
+
+Put simply, accessibility is the idea that your apps should be usable and enjoyable to as many people as possible. For some, this means that your app is easy to think through - there aren't three buttons that all have the same icon and do different things.
+
+For others, however, it means supporting keyboard-only usage of your app for those that cannot use the mouse, supporting screen-readers for blind users, and enabling other forms of assistive tech.
+
+> What's a screen-reader? What's "assistive tech"?
+
+A screen reader is a form of assistive technologies that allow users with a myriad of capabilities to use their computers.
+
+Consider; If you can't see the screen many use to access their emails, how can you still do so?
+
+The answer? A screen-reader, which reads the contents of the screen out loud, and a keyboard to navigate through the page using <kbd>Tab</kbd>.
+
+---
+
+Consider the following button:
+
+<button>Go home</button>
+
+This button might be read by a screen-reader as "Button, Go home". 
+
+Meanwhile, this list of items:
+
+- Ice Cream
+- Pizza
+
+Might read out as "List of 2 items." when the user has the list focused. Then, upon pressing <kbd>Tab</kbd>, it might read "Item one, Ice Cream".
```

Comment:
How would you focus this list? In my experience, lists don't take tab focus by default. When I click tab, it goes to the next tabbable item, which is an anchor below. For me, to access the list, I click the down arrow 

I'm using NVDA and have not tried out other screen readers, so it may be that different screen readers act differently

Reply:
Good callout. I've made it more generic by removing the `tab` mention